_rules/descriptive-link-5effbb.md

```diff
@@ -0,0 +1,292 @@
+---
+id: 5effbb
+name: Link in context is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the accessible name of a link together with its context describe its purpose.
+accessibility_requirements:
+  wcag20:2.4.4: # Link Purpose (In Context)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
```

Comment:
If you add an assumption that all links have the semantic role of link, you could say that passing this rule satisfies the SC. Not sure if that's what you want, but it wouldn't be unreasonable I think.